boards/serpente/Makefile.include

```diff
@@ -2,5 +2,3 @@ CFLAGS += -DBOOTLOADER_UF2
 
 # Include all definitions for flashing with bossa other USB
 include $(RIOTBOARD)/common/samd21-arduino-bootloader/Makefile.include
-# Include handling of serial and non-bossa programmers (if selected by user)
-include $(RIOTMAKE)/boards/sam0.inc.mk
```

Comment:
Are you sure this can be removed?

Reply:
Well this board is a pure USB board so there should not be a programmer that bossa available. This include only contains logic related to edbg, openocd and jlink. That doesn't apply here. Maybe @benpicco can confirm ?